libqtile/lazy.py

```diff
@@ -53,6 +54,8 @@ def __init__(self, call: CommandGraphCall, args: tuple, kwargs: dict) -> None:
         self._if_no_focused: bool = False
         self._layouts: set[str] = set()
         self._when_floating = True
+        self._check: bool | None = None
+        self._func: Callable | None = None
```

Comment:
Could we fully type Callable? It should be `Callable[[], bool]`.

We could probably do something like this as a default value rather than `None`, what do you think?:
```py
        self._func: Callable[[], bool] = lambda: True
```